content/_code-samples/build-a-wallet/js/vite.config.js

```diff
@@ -0,0 +1,43 @@
+import { defineConfig, loadEnv } from 'vite';
+
+import { NodeGlobalsPolyfillPlugin } from '@esbuild-plugins/node-globals-polyfill';
+import polyfillNode from 'rollup-plugin-polyfill-node';
+
+const viteCongig = ({ mode }) => {
```

Comment:
nit: Seems like this should say `viteConfig` instead. (Export then has to match)
```suggestion
const viteConfig = ({ mode }) => {
```